#asr control message

int32   order   #0：开启asr功能，1：关闭asr功能
int32   asrType #0：百度asr，1：讯飞asr